src/OrbitGl/TimeGraph.h

```diff
@@ -68,7 +68,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   [[nodiscard]] const orbit_client_model::CaptureData* GetCaptureData() const {
     return capture_data_;
   }
-  [[nodiscard]] TrackManager* GetTrackManager() { return track_manager_.get(); }
+  [[nodiscard]] TrackManager* GetTrackManager() const { return track_manager_.get(); }
```

Comment:
it should probably return const TrackManager*

Reply:
I reverted the change actually.